src/components/Header/Header.jsx

```diff
@@ -0,0 +1,21 @@
+export const Header = ({ inputChange }) => (
```

Comment:
```suggestion
export const Header = ({ onInputChange }) => (
```
In react there is a convention on how to name handlers and functions that you pass via props too:
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364